src/database/dbConnection.js

```diff
@@ -0,0 +1,8 @@
+const { Pool } = require('pg');
+require('env2')('./config.env');
+
+if (!process.env.DATABASE_URL) {
+  throw new Error('DB url not found ');
+} else {
+  module.exports = new Pool({ connectionString: process.env.DATABASE_URL });
```

Comment:
I think its error.

Reply:
if it does not work with you, you need to set up convig.env file! i think! :smile: